public/blog/testrpc-is-now-ganache.md

```diff
@@ -0,0 +1,35 @@
+# TestRPC is now Ganache
+
+Have you been looking for the TestRPC recently and haven't been able to find it?
+
+Don't worry, it's still here! It's just now known as **Ganache**.
+
+It comes in two flavors: A fully-interactive graphical interface, and the more familiar command-line version.
```

Comment:
I'd swap "It" for "Ganache".

Reply:
And maybe say instead: "A fully-interactive development blockchain with graphical interface, and the more familiar command-line version."

(up to you/artistic)